src/nvme/nbft.c

```diff
@@ -96,15 +99,17 @@ static int __get_heap_obj(struct nbft_header *header, const char *filename,
 	}
 
 	/* check that string is zero terminated correctly */
-	*output = (char *)header + obj.offset;
+	*output = (char *)header + le32_to_cpu(obj.offset);
```

Comment:
These are all major bug fixes.  How did this stuff work before these changes?

Reply:
I suppose big endian architectures are pretty rare nowadays, this still needed to be done for the sake of completeness. Same philosophical question goes for 32-bit archs.